sdk/healthinsights/pom.xml

```diff
@@ -10,5 +10,6 @@
   <modules>
       <module>azure-health-insights-cancerprofiling</module>
       <module>azure-health-insights-clinicalmatching</module>
+      <module>azure-health-insights-radiologyinsights</module>
```

Comment:
It also need to be added to ci.yml (should be at same folder).

Reply:
Added it to ci.yml with latest commit to the branch.